client/my-sites/theme/main.jsx

```diff
@@ -1464,6 +1465,9 @@ class ThemeSheet extends Component {
 				upsellNudgePlan =
 					isExternallyManagedTheme || isBundledSoftwareSet ? PLAN_BUSINESS : PLAN_PREMIUM;
 			}
+			const upsellNudgeFeature = config.isEnabled( 'themes/tiers' )
+				? `${ themeTier?.slug }-themes`
+				: WPCOM_FEATURES_PREMIUM_THEMES_UNLIMITED;
```

Comment:
This is not very pretty but does the trick.

Reply:
Great point!
Addressed (and much improved) in https://github.com/Automattic/wp-calypso/pull/87126/commits/dc359a1ec161834deb0467352ba94cfb94af7470